components/Task.js

```diff
@@ -2,29 +2,32 @@ import React from "react";
 import styled from "styled-components";
 import TaskItem from "./TaskItem";
 
+function generateID() {
+  return Math.random().toString(36).substring(2);
+}
```

Comment:
at some later stage you might create a utils.js and refactor this function there, for now its fine.